ReSharper.FSharp/test/src/FSharp.Intentions.Tests/src/QuickFixes/AddMissingSeqFixTest.fs

```diff
@@ -0,0 +1,20 @@
+namespace JetBrains.ReSharper.Plugins.FSharp.Tests.Intentions.QuickFixes
+
+open JetBrains.ReSharper.Plugins.FSharp.Psi.Features.Daemon.QuickFixes
+open JetBrains.ReSharper.Plugins.FSharp.Tests
+open NUnit.Framework
+
+[<FSharpTest>]
+type AddMissingSeqFixTest() =
+    inherit FSharpQuickFixTestBase<AddMissingSeqFix>()
+
+    override x.RelativeTestDataPath = "features/quickFixes/addMissingSeqFix"
+
+    [<Test>] member x.``FS0740 Adds missing seq before sequential`` () = x.DoNamedTest()
+    [<Test>] member x.``FS0740 Adds parens when needed app`` () = x.DoNamedTest()
+    [<Test>] member x.``FS0740 Adds parens when needed dot`` () = x.DoNamedTest()
+    [<Test>] member x.``FS0740 Adds parens when needed multiline`` () = x.DoNamedTest()
+    [<Test>] member x.``FS3873 Adds missing seq before start to finish`` () = x.DoNamedTest()
+    [<Test>] member x.``FS3873 Adds parens when needed app`` () = x.DoNamedTest()
+    [<Test>] member x.``FS3873 Adds parens when needed dot`` () = x.DoNamedTest()
+    [<Test>] member x.``FS3873 Adds parens when needed multiline`` () = x.DoNamedTest()
```

Comment:
These tests really duplicate the `FS0740` ones. Let's try to keep less duplication, please?

Reply:
Removed duplicated test